src/VisualStudio/Core/Def/Packaging/PackageInstallerServiceFactory.cs

```diff
@@ -351,7 +354,7 @@ private static string GetStatusBarText(string packageName, string installedVersi
             return false;
         }
 
-        private string GetInstalledVersion(string packageName, EnvDTE.Project dteProject)
+        private string? GetInstalledVersion(string packageName, EnvDTE.Project dteProject)
```

Comment:
Should this function just be using our cache, or is there some resaon it's still using the older API?